_blocks/home-blog-headlines/block.md

```diff
@@ -8,54 +8,22 @@ name: Blog Headlines (homepage)
 
 ## Blog
 
+{::blogs count="3"}
 <ul class="usa-card-group">
-  <li class="usa-card tablet:grid-col-4">
-    <div class="usa-card__container">
-      <div class="usa-card__header">
-        <h3 class="usa-card__heading">How one postdoc overcame imposter fears</h3>
-      </div>
-      <div class="usa-card__body">
-        <p>
-          In a Nature Career Column, Kelsey Inouye, describes her career path and her struggles along the way. She completed her PhD at the University of Oxford...
-        </p>
-      </div>
-      <div class="usa-card__footer">
-        <a href="#">read this post</a>
-      </div>
-    </div>
-  </li>
-  <li class="usa-card tablet:grid-col-4">
-    <div class="usa-card__container">
-      <div class="usa-card__header">
-        <h3 class="usa-card__heading">How one postdoc overcame imposter fears</h3>
-      </div>
-      <div class="usa-card__body">
-        <p>
-          In a Nature Career Column, Kelsey Inouye, describes her career path and her struggles along the way. She completed her PhD at the University of Oxford...
-        </p>
-      </div>
-      <div class="usa-card__footer">
-        <a href="#">read this post</a>
-      </div>
-    </div>
-  </li>
-
-  <li class="usa-card tablet:grid-col-4">
-    <div class="usa-card__container">
-      <div class="usa-card__header">
-        <h3 class="usa-card__heading">How one postdoc overcame imposter fears</h3>
-      </div>
-      <div class="usa-card__body">
-        <p>
-          In a Nature Career Column, Kelsey Inouye, describes her career path and her struggles along the way. She completed her PhD at the University of Oxford...
-        </p>
-      </div>
-      <div class="usa-card__footer">
-        <a href="#">read this post</a>
-      </div>
-    </div>
-  </li>
+  <% headlines.each do |headline| %>
```

Comment:
Yo dawg, I heard you like templating languages, so I put ERB in your Kramdown custom extension.

Reply:
Are there any security implications with putting executable code in our Markdown? I see that this only applies to the inside of the `{::blogs}` block. Probably ok?

Reply:
It does only apply inside `{::blogs}` but I'm also going to run a quick test to verify it's good. I'm pretty sure that a combination of ERB escaping things automatically and our Content Security Policy should protect us.

Reply:
Turns out the CSP was helping, but escaping was not happening. Just pushed up a change to do that explicitly